tests/TestCase/ORM/QueryRegressionTest.php

```diff
@@ -1585,8 +1585,7 @@ public function testFunctionExpressionWithSubquery(): void
                         ->ABS([
                             $table
                                 ->getConnection()
-                                ->newQuery()
-                                ->select(-1),
+                                ->newSelectQuery(-1),
```

Comment:
Since `Table` has `selectQuery()` I'm learning towards connection using the same name.

Reply:
I am fine with dropping the `new` prefix for the `Connection` methods returning a query.

Reply:
Having the same name at both layers is nice. Means developers have to learn fewer naming conventions and can more easily remember the method names, perhaps before autocomplete gets there.

Reply:
Done